lib/linter-rubocop.coffee

```diff
@@ -9,7 +9,8 @@ class LinterRubocop extends Linter
 
   # A string, list, tuple or callable that returns a string, list or tuple,
   # containing the command line (with arguments) used to lint.
-  cmd: 'rubocop --force-exclusion --format emacs'
+  cmd: atom.config.get("linter-rubocop.cmd") ||
+         'rubocop --force-exclusion --format emacs'
```

Comment:
`atom.config.get("linter-rubocop.cmd")` <-- when the value is empty or the user nulls it or something, you'll automatically get the default value. So `atom.config.get("linter-rubocop.cmd") || "value"` is redundant.